package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "politicoapp_api",
+  "version": "1.0.0",
+  "description": "An API endpoint that performs a CRUD operation for politicoApp",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon server.js --exec babel-node --"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/chokonaira/politicoApp_api.git"
+  },
+  "author": "Henry Okonkwo",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/chokonaira/politicoApp_api/issues"
+  },
+  "homepage": "https://github.com/chokonaira/politicoApp_api#readme",
+  "dependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
@chokonaira Nice work you have here! can you please use `babel-preset-env` as against the `babel-preset-es2015` because it is deprecated https://babeljs.io/docs/en/env